src/schema/city/index.ts

```diff
@@ -100,21 +104,61 @@ export const City = {
   description: "A city-based entry point for local discovery",
   args: {
     slug: {
-      type: new GraphQLNonNull(GraphQLString),
+      type: GraphQLString,
```

Comment:
Just to be sure, I'm _assuming_ that no production code was making use of this type yet and so this backwards compatibility breaking change is ok.

Reply:
That is correct. The open PR https://github.com/artsy/emission/pull/1252 is the first potential use case for this.